### DISCUSSION TITLE
Enhancement: SWF file creation or code

### DISCUSSION THREAD

**Discussion by buoya42** [FunkinCrew/Funkin#4027] [Q&A]
### Issue Checklist

- [X] I have properly named my enhancement
- [ ] I have checked the Issues/Discussions pages to see if my enhancement has already been suggested

### What is your suggestion, and why should it be implemented?

Hi, i want to use your FNF game in my website, but i don't have the code, or i don't know from where to take it (i don't want to die, tryin' to find it so...) if is possible give me the code, please or if is possible, can u make a swf file?

Sorry for the inconvenience,
buoya42

**Comment by JVNpixels** [comment]
I’m not the best with the HTML5 stuff but im pretty there is a HTML5 release zip file in the latest release of the game.

**Comment by moxie-coder** [comment]
SWF is for flash, you’re looking for html5 at most

**Comment by buoya42** [comment]
Hi, thx for the information about where can i find the html code, and I have Ruffle implementation in my website, that's why i also wanted if was possible a SWF file. Thanks equally to your information 💚!

**Comment by moxie-coder** [comment]
> Hi, thx for the information about where can i find the html code, and I have Ruffle implementation in my website, that's why i also wanted if was possible a SWF file. Thanks equally to your information 💚!

it’s not possible to get an SWF file at all, flash is dead, and I’m pretty sure this won’t compile to flash at all

**Comment by JVNpixels** [comment]
> Hi, thx for the information about where can i find the html code, and I have Ruffle implementation in my website, that's why i also wanted if was possible a SWF file. Thanks equally to your information 💚!
 
You can’t build this game to flash or ruffle, you’ll need HaxeFlixel, you should learn how to install it and use it, then compile it to a web browser.

**Comment by JVNpixels** [comment]
To upload it, after it has compiled you should zip all the contents into a .zip file or another .zip format and upload it to the site.

**Comment by JVNpixels** [comment]
Assuming it’s your website, make an uploading thing or add the ZIP somehow (this is how I port games online), I’m just speaking from experience and have only uploaded to itch.io and newgrounds.

**Comment by buoya42** [comment]
thx for all your help, this helps me a lot 💚

**Comment by JVNpixels** [comment]
You are welcome! There is no SWF or Flash file needed for this game!
